lib/nats.js

```diff
@@ -876,6 +876,17 @@ Client.prototype.processMsg = function() {
 };
 
 /**
+ * Push a new cluster server.
+ *
+ * @param {String} uri
+ * @api public
+*/
+
+Client.prototype.addServer = function(uri) {
+  this.servers.push(new Server(url.parse(uri)));
```

Comment:
What if it already exists in the list? Should we de-dupe? Should we check for any errors from parse?